config/kernel-fadvise.m4

```diff
@@ -0,0 +1,19 @@
+dnl #
+dnl # Linux 4.19 API
+dnl #
+AC_DEFUN([ZFS_AC_KERNEL_FADVISE], [
```

Comment:
This configure check should be updated to operate in parallel like all the others with a `ZFS_AC_KERNEL_FADVISE` and `ZFS_AC_KERNEL_SRC_FADVISE`